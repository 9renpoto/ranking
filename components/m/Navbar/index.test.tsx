import React from 'react'
import { create } from 'react-test-renderer'

import { Navbar } from '.'

jest.mock('../../../app/routes')

describe('Molecules.Navbar', () => {
  it('render', () => {
    const tree = create(<Navbar />)
    expect(tree.toJSON()).toMatchSnapshot()
  })
})
