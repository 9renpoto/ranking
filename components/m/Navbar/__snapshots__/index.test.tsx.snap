// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Molecules.Navbar render 1`] = `
<styled.nav>
  <ul>
    <li>
      <LinkRoutes
        params={
          Object {
            "memberId": "a27218b8-6a4d-47bb-95b6-5a55334fac1c",
            "orgId": "bb463b8b-b76c-4f6a-9726-65ab5730b69b",
          }
        }
        route="mypage"
      >
        <a>
          <FontAwesomeIcon
            border={false}
            className=""
            fixedWidth={false}
            flip={null}
            icon={
              Object {
                "icon": Array [
                  448,
                  512,
                  Array [],
                  "f007",
                  "M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z",
                ],
                "iconName": "user",
                "prefix": "fas",
              }
            }
            inverse={false}
            listItem={false}
            mask={null}
            pull={null}
            pulse={false}
            rotation={null}
            size={null}
            spin={false}
            symbol={false}
            transform={null}
          />
        </a>
      </LinkRoutes>
    </li>
  </ul>
</styled.nav>
`;
